src/main/java/seedu/address/logic/internal/nusmods/ImportMods.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.logic.internal.nusmods;
+
+import java.util.Optional;
+import java.util.logging.Logger;
+
+import org.json.simple.JSONArray;
+
+import seedu.address.commons.core.AppSettings;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.module.Module;
+import seedu.address.model.module.ModuleSummary;
+import seedu.address.model.module.ModuleSummaryList;
+import seedu.address.websocket.Cache;
+import seedu.address.websocket.NusModsApi;
+import seedu.address.websocket.NusModsParser;
+
+/**
+ * Internal class to be executed as a standalone program to import all NUSMods detailed module data.
+ */
+public class ImportMods {
+    private static final Logger logger = LogsCenter.getLogger(Cache.class);
+
+    /**
+     * Main driver.
+     */
+    public static void main(String[] args) {
```

Comment:
Can consider making this a function instead of a main method so that we can anytime add this to a client facing command instead

Reply:
Hmm makes sense, I'll do that and merge around 4.30pm